go.mod

```
@@ -16,7 +15,7 @@ require (
 	github.com/qdm12/log v0.1.0
 	github.com/qdm12/ss-server v0.4.0
 	github.com/qdm12/updated v0.0.0-20210603204757-205acfe6937e
-	github.com/stretchr/testify v1.7.2
+	github.com/stretchr/testify v1.8.0
```

let's revert this for now since it's a bit irrelevant.

```suggestion
	github.com/stretchr/testify v1.7.2
```

and then please re-run `go mod tidy`